Assets/Teak/Editor/iOS/Xcode/TeakExtensionMethods.cs

```diff
@@ -81,6 +82,29 @@ public static class TeakExtensions {
                 }
             }
         }
+
+        // From: https://stackoverflow.com/a/23697173
```

Comment:
Personal preference from painful experience: Grab this from archive.org and put the archive.org link into the comment.